{{cookiecutter.project_slug}}/.github/dependabot.yml

```
@@ -1,7 +1,90 @@
+# Config for Dependabot updates. See Documentation here:
+# https://help.github.com/github/administering-a-repository/configuration-options-for-dependency-updates
+
 version: 2
 updates:
   # Update Github actions in workflows
   - package-ecosystem: "github-actions"
     directory: "/"
+    # Check for updates to GitHub Actions every weekday
+    schedule:
+      interval: "daily"
+
+{%- if cookiecutter.use_docker == 'y' %}
+
+  # Enable version updates for Docker
+  # As there is no way to use wildcards, each dockerfile has its own entry
+  # Check https://github.com/dependabot/dependabot-core/issues/2178 for updates to that...
```

I think this comment might need a bit more clarification: one might understand it as missing support for this `compose/*/*/` and might change the `directory` to `compose/`, thinking that recursive search would work - it appears  to be working fine for `pip` - although it doesn't in the case of Docker. A bit annoying that we have to repeat all this, but it's sadly out of our control...

I think we should mention that it's not only about wildcards, but also about not being recursive. 

```suggestion
  # We need to specify each Dockerfile in a separate entry because Dependabot doesn't 
  # support widlcards or recursively checking subdirectories. Check this issue for updates:
  # https://github.com/dependabot/dependabot-core/issues/2178
```